docs/DeveloperGuide.md

```diff
@@ -399,80 +450,45 @@ We chose to implement the changing of view panels through `CommandResult` due to
 - Within in view panel class, there are similar structures (FXML file name attribute, Logger, ListView, constructor, overwriting ListCell etc.), but that is where the similarities end. We feel that having the panels inherit from a single panel does not necessarily add any value, as they do not share any attribute or methods. We find that the use of polymorphism here to make all the different view panels inherit from a single parent class is an example of over engineering and adds no value to justify the effort of doing so.
 - Only within the three `ListPanel`, there is a single common method `numRecordsString`, which checks for plurality of the word 'record'. As such, a small bit of polymorphism in an abstract class `ListPanel` is incorporated in an attempt to reduce repetition.
 
+### 3.5 Proposed Features
 
+#### 3.5.1 Multiple possible prefixes per command
 
-
-
-<!-- problematic, rmb to update the header hashs after complete -->
-
-### Edit Client and Edit Meeting feature
-
-`editClient` and `editMeeting` execute in a similar manner to each other.
-Let the term entity refer to either a client or a meeting.
-Every entity is uniquely identified by a UUID in storage, so essentially all fields in an entity can be edited without loss of its uniqueness.
-
-Below is an activity diagram that summarizes how the updates are reflected in MyInsuRec.
-EditCommand::execute() caused the model to be updated first and then the storage
-
-<img src="images/EditActivityDiagram.png" width="200" />
-
-#### Design Considerations
-
-Aspect: Class that triggers model and storage update
-
-- Alternative Solution 1 (Current choice):
-- LogicManager::execute() causes both updates
-- Pros: Any error in updating storage or model can be identified within execute()
-- Cons: execute command performs two functions, update model and update storage, which is not ideal for separating responsibilities.
-
-- Alternative Solution 2:
-- LogicManager causes Model update which internally triggers Storage update
-- Pros: Removes one instance of cohesion between Logic and Storage (Logic can access Storage via Model only)
-- Cons: Model is a single point of failure in this scheme.
-
-<img src="images/AlternativeEditActivityDiagram.png" width="250" />
-
-#### Rationale
-LogicManager is responsible for coordinating both Model and Storage updates because Model and Storage should be kept as separate entities.
-
-[Proposed] Multiple possible prefixes per command feature
-
-In this proposed feature, the user is provided with  multiple possible prefixes for defining fields in a command.
+Purpose: In this proposed feature, the user is provided with  multiple possible prefixes for defining fields in a command.
```

Comment:
Can you help me understand this sentence better?

Reply:
Can use alternative prefixes such as `d/` and `b/` for defining the same field, say the birthday field of a client. Any suggestion on how to word better?

Reply:
Sorry I still do not understand this proposed feature. And does it really add value to our product? If not, I think it is okay to delete this whole portion

Reply:
Maybe the other team members can understand and give better suggestions

Reply:
I added this section so  I could try mentioning a proposed feature, as is allowed in dg.  Not sure how to proceed either.... don't mind if PR is closed.

Reply:
Do you mean like allowing users to customize prefixes?

Reply:
we provide alternative prefixes, but an extension to this feature can be to allow users to customize their own prefixes.